### PR TITLE
Fix: Fixed session middleware and added logs to auth middleware

### PR DESCRIPTION
**Pull request by franciscog-kainos**
### JIRA link

N/A

### Change description

This PR will point to a new branch of the node session handler which contains fixes to previous bugs and basic logging.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/PenaltyDetailsController.ts [hunk 1]
```diff
@@ -55,7 +58,12 @@ export class PenaltyDetailsController extends BaseAsyncHttpController {
         const extraData = session.getExtraData();
 
         const changePenaltyIdentifier = (appeals: Appeal) => {
-            appeals[AppealKeys.PENALTY_IDENTIFIER] = body;
+            const companyNumber = appeals[AppealKeys.PENALTY_IDENTIFIER][PenaltyIdentifierKeys.COMPANY_NUMBER];
```
Comment (n-r-b) [inline src/controllers/PenaltyDetailsController.ts]: is this working for you? the company number is not being sanitised for me

Comment (franciscog-kainos) [reply src/controllers/PenaltyDetailsController.ts]: I thought this was working. I just made this pass the linting test, I don't think I changed the logic though...

Comment (n-r-b) [reply src/controllers/PenaltyDetailsController.ts]: you didn't :-) i'll take another look

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@hapi/joi": "^17.1.0",
     "body-parser": "^1.18.2",
-    "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#ee7eba7",
+    "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#hotfix/sessionHandler",
```
[the remark this answers is not in the document]
Comment (n-r-b) [reply package.json]: i don't think this branch exists?

[the remark this answers is not in the document]
Comment (franciscog-kainos) [reply package.json]: Pointing to stable commit in node-session-handler master brancg now

**Pull request merged**